src/index.js

```diff
@@ -10,12 +10,14 @@ export default function kdbush(points, getX, getY, nodeSize, ArrayType) {
 function KDBush(points, getX, getY, nodeSize, ArrayType) {
     getX = getX || defaultGetX;
     getY = getY || defaultGetY;
-    ArrayType = ArrayType || Array;
+    ArrayType = ArrayType || Float32Array;
```

Comment:
Let's use `Float64Array` here — `Float32Array` is quite lossy, and regular JS arrays use doubles under the hood.

Reply:
I've updated it to `Float64Array` :)